esp/src/eclwatch/nls/hpcc.js

```diff
@@ -560,6 +561,7 @@ define({root:
     Priority: "Priority",
     Process: "Process",
     Processes: "Processes",
+    ProcessesDown: "Processes Down",
```

Comment:
@miguelvazq does this mean its a 7.8 change?

Reply:
I fear that it may have to be due to the translation.